site/content/docs/07-state-management.md

```diff
@@ -14,7 +14,7 @@ Inside a component, get references to the stores like so:
 ```
 
 * `preloading` contains a readonly boolean value, indicating whether or not a navigation is pending
-* `page` contains a readonly `{ host, path, params, query }` object, identical to that passed to `preload` functions
+* `page` contains a readonly `{ host, path, params, query, error, status }` object. The same value is passed to `preload` functions, though `status` is not present at that time and `error` is only set on error pages.
```

Comment:
I'm wondering now if my suggestion to add `status` maybe wasn't a good one. I hadn't thought through the fact that it wouldn't be available yet in `preload`. I wonder if it's that useful in this case and maybe that's too much of a caveat to actually add it

Reply:
Yeah, I wouldn't mind removing it again. I generally don't find it particularly useful and the concept of an http status code doesn't really make sense on the client side. I'll take it out again.